installers/flash-tarball/radiasoft-download.sh

```diff
@@ -63,9 +65,15 @@ flash_tarball_patch_and_update_tgz() {
 > F90FLAGS = -fallow-argument-mismatch
 EOF
     cd ..
```

Comment:
This assumes that `cd "$1"` is only one level. Better to cd "$p" here and set p=$PWD at the top.

Reply:
d146ff4